src/libraries/Common/src/System/Net/Security/NetEventSource.Security.Windows.cs

```diff
@@ -9,87 +9,178 @@ namespace System.Net
 {
     internal sealed partial class NetEventSource
     {
+        private const int EnumerateSecurityPackagesId = NextAvailableEventId + 1;
```

Comment:
Why is it +1, isn't NextAvailableEventId already available? 😄

Reply:
Good question 😆
It doesn't need the +1 here. I'll update that